detox/package.json

```diff
@@ -81,6 +82,10 @@
         "functions": 100,
         "lines": 100
       }
-    }
+    },
```

Comment:
Not sure excluding `src/index.js` is the right thing. This is something we can test, it's actually important that we do, till we have a sane type system, we want to make sure beforeEach and afterEach exist and that they don't crash the test if detox is undefined.